app/Http/Controllers/AuthorController.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\AuthorRequest;
+use App\Repositories\RepositoryInterface\BaseRepositoryInterface;
+
+class AuthorController extends Controller
+{
+    protected $authorRepository;
+
+    public function __construct(BaseRepositoryInterface $authorRepository){
+        $this->authorRepository = $authorRepository;
+    }
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */ 
+    public function index()
+    {
+        $author = $this->authorRepository->getAuthor();
```

Comment:
```suggestion
        $authors = $this->authorRepository->getAuthor();
```
nếu trra về nhiều thì đặt tên số nhiều, bên view mình đặt tên trong for cho đúng nhé

Reply:
dạ em hiểu rồi